Iryna_Ramanets/Module_4_Framework/M4_HW1/src/test/java/com/epam/gmail/test/spam/TestSpam.java

```diff
@@ -0,0 +1,84 @@
+package com.epam.gmail.test.spam;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import com.epam.gmail.test.module.dialog.WriteEmailDialog;
+import com.epam.gmail.test.module.page.SighInPage;
+import com.epam.gmail.test.module.page.SpamPage;
+import com.epam.gmail.test.module.toolbar.MainToolbar;
+import com.epam.gmail.test.util.Util;
+
+public class TestSpam {
+
+	private static final String URL = "http://gmail.com/";
+
+	private static final String USER1 = "TestIrRam1@gmail.com";
+	
+	private static final String USER2 = "TestIrRam2@gmail.com";
+	
+	private static final String PSWD = "IR12345678";
+	
+	private WebDriver driver;
+	
+	@BeforeClass
+	public void beforeClass() {
+		if(System.getProperty("webdriver.chrome.driver") == null) {
+			System.setProperty("webdriver.chrome.driver", "./drivers/chromedriver.exe");
+		}
+		// Create a new instance of the Chrome driver

```

Comment:
What the reason to use comments and increase amount of code lines? Your code readable enough. Honestly comments it's not a good practice.